public/app/features/explore/Logs/Logs.tsx

```diff
@@ -203,6 +215,49 @@ class UnthemedLogs extends PureComponent<Props, State> {
     }
   }
 
+  registerLogLevelsWithContentOutline = () => {
```

Comment:
I haven't checked out this locally, but did you check if this will be updated when we run a different query or change the. current query?

Reply:
Confirmed if you have a query with `logfmt` that has the filters in the outline, then remove `logfmt` and re-run the query, the filters disappear